sunpy/coordinates/transformations.py

```diff
@@ -171,26 +171,57 @@ def hcc_to_hgs(helioccoord, heliogframe):
     return heliogframe.realize_frame(representation)
 
 
+@frame_transform_graph.transform(DynamicMatrixTransform, Heliocentric,
+                                 HeliographicStonyhurst)
+def hcc_to_hgs(hcc_coord, hgs_frame):
+    """
+    Convert from Heliocentric Cartesian to Heliographic Stonyhurst.
+    """
+    if not isinstance(hcc_coord.observer, BaseCoordinateFrame):
+        raise ConvertError("Cannot transform heliocentric coordinates to "
+                           "heliographic coordinates for observer '{}' "
+                           "without `obstime` being specified.".format(hcc_coord.observer))
+
+    # Get observer longitude/lattitude with respect to Earth
+    l0 = np.deg2rad(hcc_coord.observer.lon)
+    b0 = np.deg2rad(hcc_coord.observer.lat)
```

Comment:
The `lon` and `lat` from the `hcc_coord` doesn't has already units? If so, then you don't need to convert it to radians.